### ISSUE TITLE
a11y, Editor toolbar has buttons without labels.

### ISSUE THREAD

**Issue by blindgineer** [calcom/cal.com#20227]
### Issue Summary

Text editor toolbar buttons are no labelled, so people that use screen readers don't know what are these buttons do.
It occurs in different places, such as:
1. Settings > profile > abbout.
2. Event types > any event type > event setup > description.

### Steps to Reproduce

1. [Login](https://app.cal.com/auth/login) with your account.
2. Go to one of those prior mentioned places.
3. Check that the buttons in the editor bar are not being labeled using one of the following methods.
    1. Inspect one of those buttons and observe that it doesn't have a label.
    2. Use Lighthouse developer tool on google chrome, perform an accessibility audit and show the results.

### Actual Results

- Screen readers cant notify the people about the behaviour of editor buttons.

### Expected Results

- Editor toolbar buttons should have a label so that people using screen readers are informed of the button's purpose.

### Technical details

This was tested under MacOS Sequoia 15.3.1 on the following conditions: 
- Chrome V.134.0.6998.89.
- Emulated Desktop with Lighthouse 12.3.0.
- Safari V. 18.3 (20620.2.4.11.5)

### Evidence

Check the Steps to Reproduce paragraph.

**Comment by athaulrehman0304** [comment]
@blindgineer I'm trying to locate the code for the editor toolbar buttons to work on the accessibility issue, but I'm having trouble finding it. Could someone point me to the file where the toolbar is defined?
